framework/src/org/checkerframework/framework/util/FieldInvariantObject.java

```diff
@@ -13,7 +13,9 @@
  * this field invariant applies.
  */
 public class FieldInvariantObject {
```

Comment:
@wmdietl said:
>'Object' is terrible in a class name. Is there nothing more specific we can use?

What would you suggest? FieldInvariant would be the best name, but that's the name of the annotation.